structures/ImageEmbedCommand.js

```diff
@@ -15,6 +15,7 @@ class ImageEmbedCommand extends Command {
 	 * the max number of the file name 
 	 */
 	constructor(handler, options, baseURLOrArray, maxNumber) {
+		options.clientPermissions = ['EMBED_LINKS'];
```

Comment:
Something like this would also allow you to specify own custom permissions and just add EMBED_LINKS then instead of simply overwriting it.
```js
if (options.clientPermissions) {
    options.clientPermissions.push('EMBED_LINKS');
} else {
    options.clientPermissions = ['EMBED_LINKS'];
}
```

Reply:
kek

Reply:
Will do